tests/rpc/rpc_03-chain_test.go

```diff
@@ -118,15 +118,10 @@ func TestChainRPC(t *testing.T) {
 	blockNumber, err = common.HexToUint(header.Number)
 	require.NoError(t, err)
 	assert.GreaterOrEqual(t, blockNumber, uint(1))
-
-	var blockHash string
-	fetchWithTimeout(ctx, t, "chain_getBlockHash", "[]", &blockHash)
-	assert.Regexp(t, regex32BytesHex, blockHash)
-	assert.NotEqual(t, finalizedHead, blockHash)
```

Comment:
Why is this removed?

Reply:
Put it back, I removed for a moment and I forget to rollback